src/Appender/FileAppender.cs

```diff
@@ -620,22 +620,23 @@ public class MinimalLock : LockingModelBase
 			private string m_filename;
 			private bool m_append;
 			private Stream m_stream = null;
-
-			/// <summary>
-			/// Prepares to open the file when the first message is logged.
-			/// </summary>
-			/// <param name="filename">The filename to use</param>
-			/// <param name="append">Whether to append to the file, or overwrite</param>
-			/// <param name="encoding">The encoding to use</param>
-			/// <remarks>
-			/// <para>
-			/// Open the file specified and prepare for logging.
-			/// No writes will be made until <see cref="AcquireLock"/> is called.
-			/// Must be called before any calls to <see cref="AcquireLock"/>,
-			/// <see cref="ReleaseLock"/> and <see cref="CloseFile"/>.
-			/// </para>
-			/// </remarks>
-			public override void OpenFile(string filename, bool append, Encoding encoding)
+            private Mutex m_appendMutex = null;
+            private string m_appendMutexFriendlyName = "MUTEX_INTERPROCESS_COMMUNICATION_ACROSS_PROCESSES";
```

Comment:
Should this be a more random string? `MUTEX_INTERPROCESS_COMMUNICATION_ACROSS_PROCESSES` might be in use by anyone. The mutex name should at least contain `log4net` and a scope what it is used for. Interprocess communication is not what the mutex locks and thus not the correct scope.